packages/wallet/test/services/DRepRegistrationTracker.test.ts

```diff
@@ -0,0 +1,79 @@
+import { Cardano } from '@cardano-sdk/core';
+import { coldObservableProvider } from '@cardano-sdk/util-rxjs';
+import { createDRepRegistrationTracker } from '../../src';
+import { createTestScheduler } from '@cardano-sdk/util-dev';
+
+jest.mock('@cardano-sdk/util-rxjs', () => {
+  const actual = jest.requireActual('@cardano-sdk/util-rxjs');
+  return {
+    ...actual,
+    coldObservableProvider: jest.fn().mockImplementation((...args) => actual.coldObservableProvider(...args))
+  };
+});
+
+describe('createDRepRegistrationTracker', () => {
+  const coldObservableProviderMock = coldObservableProvider as jest.MockedFunction<typeof coldObservableProvider>;
```

Comment:
Where is this used?

Reply:
removed

Reply:
Then `jest.mock('@cardano-sdk/util-rxjs', () => {` is also not needed

Reply:
resolved in [79d58ead95895002e4f62e8ed9e002a26740bbe8](https://github.com/input-output-hk/cardano-js-sdk/pull/1192/commits/79d58ead95895002e4f62e8ed9e002a26740bbe8)